compass/ocean/tests/utility/combine_topo/__init__.py

```diff
@@ -158,169 +176,411 @@ def _modify_bedmachine(self):
+            '+k_0=1.0 +x_0=0.0 +y_0=0.0 +ellps=WGS84'
+        )
+
+        # Create SCRIP file using pyremap
+        bedmachine_descriptor = ProjectionGridDescriptor.read(
+            projection, in_filename, 'BedMachineAntarctica500m',
+        )
+        bedmachine_descriptor.to_scrip(out_filename)
+
+    def _create_target_scrip_file(self):
+        """
+        Create SCRIP file for either the x.xxxx degree (lat-lon) mesh or the
+        NExxx (cubed-sphere) mesh, depending on the value of `self.target_grid`
+        References:
+          https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/872579110/
+          Running+E3SM+on+New+Atmosphere+Grids
+        """
+        logger = self.logger
+        logger.info(f'Create SCRIP file for {self.resolution_name} mesh')
+
+        out_filename = f'{self.resolution_name}.scrip.nc'
+
+        # Build cubed sphere SCRIP file using tempestremap
+        if self.target_grid == 'cubed_sphere':
+
+            # Create EXODUS file
+            args = [
+                'GenerateCSMesh', '--alt', '--res', f'{self.resolution}',
+                '--file', f'{self.resolution_name}.g',
+            ]
+            logger.info(f'    Running: {" ".join(args)}')
+            subprocess.run(args, check=True)
+
+            # Create SCRIP file
+            args = [
+                'ConvertMeshToSCRIP', '--in', f'{self.resolution_name}.g',
+                '--out', out_filename,
+            ]
+            logger.info(f'    Running: {" ".join(args)}')
+            subprocess.run(args, check=True)
+
+        # Build lat-lon SCRIP file using pyremap
+        elif self.target_grid == 'lat_lon':
+            descriptor = get_lat_lon_descriptor(
+                dLon=self.resolution, dLat=self.resolution,
+            )
+            descriptor.to_scrip(out_filename)
 
-        projection = pyproj.Proj('+proj=stere +lat_ts=-71.0 +lat_0=-90 '
-                                 '+lon_0=0.0 +k_0=1.0 +x_0=0.0 +y_0=0.0 '
-                                 '+ellps=WGS84')
+        logger.info('  Done.')
 
-        bedmachine = xr.open_dataset(in_filename)
-        x = bedmachine.x.values
-        y = bedmachine.y.values
+    def _create_weights(self, in_filename, out_filename):
+        """
+        Create weights file for remapping to `self.target_grid`. Filenames
+        are passed as parameters so that the function can be applied to
+        GEBCO and BedMachine
 
-        in_descriptor = ProjectionGridDescriptor.create(
-            projection, x, y, 'BedMachineAntarctica_500m')
+        Parameters
+        ----------
+        in_filename : `str`, source file name
+        out_filename : `str`, weights file name
+        """
+        logger = self.logger
 
-        out_descriptor = LatLonGridDescriptor.read(fileName=gebco_filename)
+        config = self.config
+        method = config.get('combine_topo', 'method')
+
+        # Generate weights file
+        args = [
+            'srun', '-n', f'{self.ntasks}',
```

Comment:
We want to let compass take care of this, rather than hard-coding `srun` and its flags.  The preferred way to do this it to call `compass.parallal.run_command()` something like the way it's done for the model:
https://github.com/MPAS-Dev/compass/blob/2e06f2e0452cc7dd410bc47a5374f90ed34ea169/compass/model.py#L60